### PR TITLE
chore: add helm chart values for base url

### PR DESCRIPTION
**Pull request by whitneypurdum**
https://energyweb.atlassian.net/browse/IVA-13 
- Add BASE_URL value to helm charts

### CODE REVIEW

File: devops/ssi-helm/templates/deployment.yaml [hunk 1]
```diff
@@ -45,6 +45,11 @@ spec:
           #   httpGet:
           #     path: /
           #     port: http
+          env:
+            {{- range $key, $value := .Values.appValues }}
+            - name: {{ $key }}
+              value: {{ $value | quote }}
+            {{- end }}
```
Comment (jrhender) [inline devops/ssi-helm/templates/deployment.yaml:52]: Is this a common pattern for helm charts?

What does the `quote` value mean?

**Review by jrhender — COMMENTED** [review]
Thanks for the PR @whitneypurdum . Just a couple questions

File: devops/ssi-helm/templates/deployment.yaml (hunk 1 above)
Comment (whitneypurdum) [reply devops/ssi-helm/templates/deployment.yaml:52]: I pulled this pattern from what we did in rebeam. The quotes are a safety check for referencing strings in helm charts I believe: https://helm.sh/docs/howto/charts_tips_and_tricks/#quote-strings-dont-quote-integers

Comment (whitneypurdum) [reply devops/ssi-helm/templates/deployment.yaml:52]: https://github.com/energywebfoundation/rebeam-helm/blob/f826c617961598d4cafd57af58d497b711ed071f/templates/deployment.yaml#L50

Comment (jrhender) [reply devops/ssi-helm/templates/deployment.yaml:52]: > I pulled this pattern from what we did in rebeam. The quotes are a safety check for referencing strings in helm charts I believe: https://helm.sh/docs/howto/charts_tips_and_tricks/#quote-strings-dont-quote-integers

Ah, I see, it's a pipe into a `quote` function. Looks good to me, thanks for the explanation!

**Pull request merged**